hudi-common/src/main/java/org/apache/hudi/common/table/timeline/HoodieActiveTimeline.java

```diff
@@ -795,6 +822,22 @@ protected void createFileInMetaPath(String filename, Option<byte[]> content, boo
       metaClient.getFs().createImmutableFileInPath(fullPath, content);
     }
   }
+  
+  protected void createCompleteInstantWithCompletionTime(HoodieInstant instant, Option<byte[]> content) {
+    try {
+      trueTimeGenerator.lock();
+      String currentTime = HoodieInstantTimeGenerator.formatDate(new Date(trueTimeGenerator.nowAsMillis(false)));
+      HoodieInstant instantWithCompletionTime = new HoodieInstant(instant.getState(), instant.getAction(),
```

Comment:
Can you explain why the `HoodieActiveTimeline#createNewInstantTime` can not work correctly here?